templates/default/serverspec.erb

```diff
@@ -0,0 +1,7 @@
+require 'serverspec'
+
+set :backend, :exec
+
+describe file('/') do
```

Comment:
Should we actually have a default test in here or does it fail if there isn't one?